test/helpers/apply/apply.go

```diff
@@ -30,8 +32,44 @@ import (
 
 var decUnstructured = yaml.NewDecodingSerializer(unstructured.UnstructuredJSONScheme)
 
-// Apply applies a kustomization to the cluster using the given rest.Config.
-func Apply(ctx context.Context, t *testing.T, cfg *rest.Config, dir string) {
+// Template applies templated resources to the cluster using the given rest.Config.
+func Template(t *testing.T, cfg *rest.Config, chartPath string, templates []string) {
```

Comment:
I _think_ the change in these package is OK, going from tool oriented to verb oriented but I'm concerned that down the line when we want to add new actions we'd require a new package vs in the old package structure, we'd only need a new func. WDYT?

I believe though that in this context we'd want to rename this function and `templateRenderer` below so that they clearly mention they should be use with helm. WDYT?